client/landing/jetpack-cloud/components/sidebar/menu.tsx

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Lets add a readme for this. 

When should this be used vs the SidebarMenu component?